### PR TITLE
Cleaned up project organization

### PR DESCRIPTION
**Pull request by Anequit**
I moved all the projects into a more standard src folder. This helps keep things organized.

### CODE REVIEW

File: src/cyberdropDownloader/cyberdropDownloader.csproj [hunk 1]
```diff
@@ -57,17 +57,23 @@
...
+    <SignManifests>false</SignManifests>
   </PropertyGroup>
   <PropertyGroup>
     <ApplicationIcon>duck.ico</ApplicationIcon>
   </PropertyGroup>
   <PropertyGroup>
     <SignAssembly>false</SignAssembly>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetZone>LocalIntranet</TargetZone>
```
Comment (izqalan) [inline src/cyberdropDownloader/cyberdropDownloader.csproj:72]: what this do?

File: src/cyberdropDownloader/Properties/app.manifest [hunk 2]
```diff
@@ -0,0 +1,73 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (izqalan) [inline src/cyberdropDownloader/Properties/app.manifest:1]: is this automatically generated?

File: src/cyberdropDownloader/cyberdropDownloader.csproj (hunk 1 above)
Comment (Anequit) [reply src/cyberdropDownloader/cyberdropDownloader.csproj:72]: I didn't add that, though it most likely has to do with the fact i had to disable clickonce in order to buid

File: src/cyberdropDownloader/Properties/app.manifest (hunk 2 above)
Comment (Anequit) [reply src/cyberdropDownloader/Properties/app.manifest:1]: Yeah

File: src/cyberdropDownloader/cyberdropDownloader.csproj (hunk 1 above)
Comment (Anequit) [reply src/cyberdropDownloader/cyberdropDownloader.csproj:72]: > I didn't add that, though it most likely has to do with the fact i had to disable clickonce in order to buid

to add to this, I didn't touch source other than reorganizing the files. I did do a test build which required me disabling clickonce and removing the temp key, but other than that nothing else.

**Comment by izqalan** [comment]
lgtm, I'm merging this pr.

**Pull request merged**